### PR TITLE
Rspec for views: courses, user courses, course_subjects

### PR DESCRIPTION
**Pull request by QuangVinh93**

### CODE REVIEW

**Comment by hitshootnum1** [comment]
ok

File: bin/rails [hunk 1]
```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env ruby
+if ENV['RAILS_ENV'] == 'test'
```
Comment (dieunb) [inline bin/rails]: double quote

File: spec/rails_helper.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# This file is copied to spec/ when you run 'rails generate rspec:install'
+ENV['RAILS_ENV'] ||= 'test'
```
Comment (dieunb) [inline spec/rails_helper.rb]: double quote

File: spec/rails_helper.rb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+require File.expand_path('../../config/environment', __FILE__)
```
Comment (dieunb) [inline spec/rails_helper.rb]: double quote

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+# Prevent database truncation if the environment is production
+abort("The Rails environment is running in production mode!") if Rails.env.production?
+require 'spec_helper'
```
Comment (dieunb) [inline spec/rails_helper.rb]: double quote

File: spec/rails_helper.rb [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+require 'rspec/rails'
```
Comment (dieunb) [inline spec/rails_helper.rb]: double quote

**Pull request merged**